loadtest/store/memstore/store.go

```diff
@@ -142,6 +142,19 @@ func (s *MemStore) SetTeamMember(teamId string, teamMember *model.TeamMember) er
 	return nil
 }
 
+
+func (s *MemStore) SetTeamMembers(teamId string, teamMembers []*model.TeamMember) error {
+	if s.teamMembers[teamId] == nil {
+		s.teamMembers[teamId] = map[string]*model.TeamMember{}
+	}
```

Comment:
The idea here is that we always want to create a new map otherwise the behaviour is the same as before. We want to replace what we have with what we get. You can take a look at `SetChannelMembers`.